style.css

```diff
@@ -20,6 +20,174 @@ body {
   overflow-x: hidden;
 }
 
+/* ----------------------------------------------POP UP display ----------------------------------------------------------- */
+.popup-container {
+  background: #fff;
+  border: 1px solid #ebecf0;
+  position: absolute;
+  z-index: 10;
+  left: 20%;
+  top: 0;
+  display: none;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 16px;
+}
+
+.popup-content {
+  font-family: "Poppins", sans-serif;
+}
+
+.headline-popup {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  margin-left: 24px;
+  width: 95%;
+  height: 40px;
+}
```

Comment:
- [ ] As requested by the previous code reviewer, the style of the pop-ups on desktop and the mobile view does not look like the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as I describe in the video below :point_down: 


https://user-images.githubusercontent.com/92303891/210893821-f15e3a24-9395-4c87-ab7c-7d80dfcd9be1.mp4